src/data_structures/annot_map.rs

```diff
@@ -45,36 +65,27 @@ impl<R, T> AnnotMap<R, T>
 where
     R: Eq + Hash,
 {
-    /// Creates a new, empty `AnnotMap`.
+    /// Create a new, empty `AnnotMap`.
     ///
     /// ```
-    /// extern crate bio;
     /// use bio::data_structures::annot_map::AnnotMap;
     /// let mut genes: AnnotMap<String,String> = AnnotMap::new();
     /// ```
     pub fn new() -> Self {
         Default::default()
     }
 
-    /// Inserts an object into the container at a specified location.
-    ///
-    /// # Arguments
-    ///
-    /// `data` is the data item to be inserted into the annotation map
-    ///
-    /// `location` is the location associated with the data item.
```

Comment:
Maybe we can keep the `Arguments` description here and in the functions below. I think that standard syntax would be something like:
```
/// * `data` - data to be inserted into the annotation map
```
And I'm assuming you skipped them, because they don't seem very useful. Maybe they could be when improved? Eventually, I don't know and if you have already put some thought into it and concluded there is no use for them, don't worry about it...

Reply:
I don't think the current descriptions are very helpful and I couldn't think of descriptions that would meaningfully improve what is provided by the parameter types in the function signatures, hence the deletions. If you think they're valuable I can put them back.

Reply:
Yeah, you're probably right. So let's leave them out for now.